kythe/typescript/indexer.ts

```diff
@@ -1957,14 +1996,16 @@ function main(argv: string[]) {
 
   // This program merely demonstrates the API, so use a fake corpus/root/etc.
   const compilationUnit: VName = {
-    corpus: 'corpus',
+    corpus: process.env.KYTHE_CORPUS || 'corpus',
```

Comment:
This should come from the compilation unit required_input VName for the source file(s).